packages/app-content-pages/src/screens/Team/TeamContainer.js

```diff
@@ -30,8 +31,9 @@ function TeamContainer (props) {
 TeamContainer.getInitialProps = async ({ req }) => {
   const host = getHost(req)
   let error
-  const teamData = await request.get(host + '/api/team')
-    .then(res => res.body)
+  let teamData
+  await request.get(host + '/api/team')
+    .then(res => teamData = res.body)
```

Comment:
Not a blocking comment but the linter should be warning about returning assignment here. Good practice is to return an actual value or wrap your function so that it returns nothing.

Reply:
intruging - only linter errors I got from `yarn lint` were about return values from fat arrow functions.

Reply:
nope - i modified the code to remove that....master linter errors are there `Arrow function should not return assignment`

Reply:
That's the one. Here you are returning `teamData = res.body`.